roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -148,12 +148,48 @@
     - not upgrade_cluster_setup
     - kubeadm_already_run.stat.exists
 
+- name: Create kubeadm token for joining nodes with 24h expiration (default)
```

Comment:
Why do we need this token in addition to the join tokens already created by `kubeadm init`?